src/main/java/co/rsk/federate/signing/hsm/message/AdvanceBlockchainMessage.java

```diff
@@ -26,10 +28,24 @@ public List<String> getParsedBlockHeaders() {
     }
 
     public String[] getParsedBrothers(String blockHeader) throws HSMBlockchainBookkeepingRelatedException {
-        return this.parsedHeaders.stream()
+        String[] parsedBrothers = this.parsedHeaders.stream()
             .filter(header -> header.getBlockHeader().equals(blockHeader))
             .findFirst()
             .map(ParsedHeader::getBrothers)
             .orElseThrow(() -> new HSMBlockchainBookkeepingRelatedException("Error while trying to get brothers for block header. Could not find header: " + blockHeader));
+        Arrays.sort(parsedBrothers);
+        return parsedBrothers;
+    }
+
+    protected List<BlockHeader> filterBrothers(List<BlockHeader> brothers) {
```

Comment:
Why protected?

Reply:
It Was Also Used In The Tests, But That Has Been Updated Too